cloud2sql/__main__.py

```diff
@@ -36,9 +39,29 @@ def parse_args() -> Namespace:
         "--analytics-opt-out",
         default=False,
         action="store_true",
-        help="Do not send anonimized analytics data",
+        help="Do not send anonymized analytics data",
+    )
+    parser.add_argument(
+        "--version",
+        default=False,
+        action="store_true",
+        help="Print version and exit",
+    )
+    parser.add_argument(
+        "--create-config",
+        action="store_true",
+        help="Print empty configuration and exit. Use this to create your own config file.",
     )
     args = parser.parse_args()
+    if args.version:
+        print(f"Cloud2SQL Version {__version__}")
+        exit(0)
+    elif args.create_config:
+        cfg = {"sources": default_config(), "destinations": {"sqlite": {"database": "cloud2sql.db"}}}
```

Comment:
Does this really help? The card I created for this was more along the lines of an interactive setup wizard asking the user what kind of clouds and databases they want to configure.

Reply:
Does not require the user to get to our repo. So yes, I guess there is still room for improvement.